### DISCUSSION TITLE
恳请退款。

### DISCUSSION THREAD

**Discussion by ggscau** [getlantern/lantern-forum-cn#1057] [General]
您好！申请专业账户邮箱  scaugg@gmail.com。申请了年度计划共计三次。付款三笔。但成功扣款，专业用户的激活码一直无法收到。只能恳请退还该三笔款。感谢！
第一笔通过支付宝扣费，第二笔和第三笔通过谷歌商店扣费。
![Screenshot_20241008_003316](https://github.com/user-attachments/assets/9568a746-8d7d-4d23-ad7c-178ca1fee8a7)
![Screenshot_20241010_134708](https://github.com/user-attachments/assets/d4e5b799-3728-490e-a4f0-907b4ce9774d)
![Screenshot_20241010_134727](https://github.com/user-attachments/assets/7da451c2-3017-45bb-b4fb-400cc533c926)

**Comment by MaxMKMax** [comment]
@ggscau 您好. 我帮您申请其他 2 年的退款可以吗? 然后 1 年的留下来. 激活码我让技术团队给您弄.

**Reply by ggscau** [reply]
管理员，您好。您的方案可行。请帮忙联系解决。非常感谢！

**Comment by ggscau** [comment]
好的。谢谢您的帮助！

**Comment by ggscau** [comment]
您好！好的。非常感谢您的帮助！

**Comment by ggscau** [comment]
@Mike777Khoo 您好！能否帮忙催促。已经四个月了。感谢！

**Comment by david-lant** [comment]
您好，关于账户的事情，请通过App，[在线表格](https://support.lantern.io/)或者邮箱[support@getlantern.org](mailto:support@getlantern.org)联系我们。请说明已经很久了没有解决，谢谢。

---

**我们将在12/24停止使用论坛或者Issues回答问题。**

如果您有问题的话，请使用以下通道解决：
- (推荐）App中的提交问题（因为我们可以看到诊断报告）
- [在线表格](https://support.lantern.io/)
- 邮箱[support@getlantern.org](mailto:support@getlantern.org)
  
如果您通过在线表格或邮箱提交工单，麻烦参照[这里](https://github.com/getlantern/guide/wiki/Troubleshooting#log-retrieval)附件诊断日志log。网络或App问题的话，我们需要诊断log才能解决。
